api/envoy/api/v2/cds.proto

```diff
@@ -385,6 +385,13 @@ message Cluster {
     // going to an individual locality if said locality is disproportionally affected by the
     // subset predicate.
     bool scale_locality_weight = 5;
+
+    // If true, when a fallback policy is configured and its corresponding subset fails to find
+    // a host this will cause any host to be selected instead.
+    //
+    // This is useful when using the default subset as the fallback policy, given the default
+    // subset might become empty which would cause requests to fail unless this option is enabled.
```

Comment:
Being very pedantic here, even if you hit panic mode here you won't be guaranteed that the request won't fail. Both the cluster being empty or all hosts being unhealthy (with lb panic mode being disabled) can cause the request to still fail. Maybe word word it along the lines of "will attempt to select a host from the entire cluster"?

Reply:
Done.